frontend/src/metabase/collections/components/BaseItemsTable.styled.jsx

```diff
@@ -34,10 +37,23 @@ export const ColumnHeader = styled.th`
   color: ${color("text-medium")};
 `;
 
+export const LastEditedByCol = styled.col`
+  width: 140px;
+
+  ${breakpointMinLarge} {
+    width: 240px;
+  }
```

Comment:
Make the column bigger on large screen to avoid seeing ellipsis

#### Small screen
![image](https://user-images.githubusercontent.com/1937582/174274329-aef61b1f-bb7b-49a9-8478-721633faef0e.png)

#### Large screen
![image](https://user-images.githubusercontent.com/1937582/174275055-558c3b11-faa0-4e0e-ae8c-2c55e866a934.png)